### ISSUE TITLE
Deprecation warning in Rails 6.0.0.rc1 for key attribute in StringTranslation, TextTranslation

### ISSUE THREAD

**Issue by morozRed** [shioyama/mobility#332]
## Context
```
mobility (0.8.7)
rails (6.0.0rc1)
```

During specs running it throws warnings

## Expected Behavior
No deprecation warnings

## Actual Behavior
```console
DEPRECATION WARNING: Uniqueness validator will no longer enforce case sensitive comparison in Rails 6.1. To continue case sensitive comparison on the :key attribute in Mobility::ActiveRecord::TextTranslation model, pass `case_sensitive: true` option explicitly to the uniqueness validator. (called from block (2 levels) in <main>

DEPRECATION WARNING: Uniqueness validator will no longer enforce case sensitive comparison in Rails 6.1. To continue case sensitive comparison on the :key attribute in Mobility::ActiveRecord::StringTranslation model, pass `case_sensitive: true` option explicitly to the uniqueness validator. (called from block (2 levels) in <main> 
```
## Possible Fix
```diff
# lib/mobility/active_record/translation.rb
validates :key, presence: true, uniqueness: {
	scope: [:translatable_id, :translatable_type, :locale],
+	case_sensitive: true
}
```

**Comment by shioyama** [comment]
Hmm, thanks, looks like an easy one to fix. If you want to take a shot at it please go ahead! #323 also needs to be fixed for Rails 6 and that one is not so easy...

**Comment by morozRed** [comment]
Sure, I'll fix it

**Comment by redross** [comment]
This doesn't seem to be released yet, right? (I don't see it in v0.8.9)
